packages/create-app/templates/default-app/packages/backend/package.json.hbs

```diff
@@ -39,7 +39,8 @@
     "express": "^4.17.1",
     "express-promise-router": "^4.1.0",
     "pg": "^8.3.0",
-    "winston": "^3.2.1"
+    "winston": "^3.2.1",
+    "node-gyp": "^9.4.0"
```

Comment:
Sorry, I missed in the last conversation that you wanted to add this to the backend `package.json` and not the root `package.json`. Feel like this is a bit mixed, we have it in the `devDependencies` in the root `package.json` and then here as a `dependencies`. I'm honestly not sure the right place though 🤔

Reply:
We want it if `devDependencies` if possible, I'd expect that to work. Let's also make sure we use the same version query as the one in the root: `^9.0.0`